controller_coverage.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Usage: ./controller_coverage.sh
+# Then run: firefox controller_coverage/index.html
+
+set -e
+set -x
+
+cd "$(dirname "$0")"
+ENVIRONMENT=native
+OUTPUT_DIR=controller_coverage
+SRC_DIR=".pio/build/$ENVIRONMENT"
+
+# delete coverage files
+#find .pio -name "*.gcno" -delete
+
+# run tests
+pio test -e native
+
+
+# delete the old report as "safely" as possible
+find "$OUTPUT_DIR" -name "*.html" -delete
```

Comment:
The script fails on this line if such directory does not already exist, e.g. on the first run.